packages/grafana-ui/src/components/ColorPicker/ColorPicker.story.tsx

```diff
@@ -1,23 +1,31 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import { boolean } from '@storybook/addon-knobs';
 import { SeriesColorPicker, ColorPicker } from './ColorPicker';
 import { action } from '@storybook/addon-actions';
 import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
 import { UseState } from '../../utils/storybook/UseState';
 import { renderComponentWithTheme } from '../../utils/storybook/withTheme';
+import mdx from './ColorPicker.mdx';
 
 const getColorPickerKnobs = () => {
   return {
     enableNamedColors: boolean('Enable named colors', false),
   };
 };
 
-const ColorPickerStories = storiesOf('Pickers and Editors/ColorPicker/Pickers', module);
-
-ColorPickerStories.addDecorator(withCenteredStory);
+export default {
+  title: 'Pickers and Editors/ColorPicker',
+  component: ColorPicker,
+  subcomponents: { SeriesColorPicker },
```

Comment:
what does subcomponent do?

Reply:
You get more components for the same Prop-table 😄 
![image](https://user-images.githubusercontent.com/1438972/80504863-e2f25c80-8973-11ea-8c24-ee7bea8c0d38.png)